server/Arcadia.Assistant.Calendar/SickLeave/EmployeeSickLeaveActor.cs

```diff
@@ -188,6 +212,20 @@ private void OnSickLeaveRequest(SickLeaveIsRequested message)
             this.ApprovalsByEvent[message.EventId] = new List<Approval>();
         }
 
+        private void OnSickLeaveDatesEdit(SickLeaveDatesAreEdited message)
```

Comment:
we do have Prolong. Basically that's what should be allowed. Simple dates changes - no

Reply:
In fact, every employee is just a people, and he can do a mistake. Will he really have to cancel existing event and create new? It is absurdity.

Reply:
This is allowed ONLY if calendar event is in requested status. So, mostly it means that event was just created.

Reply:
Reverted.